.github/workflows/linux.yml

```diff
@@ -12,9 +12,11 @@ env:
   CMAKE_BUILD_PARALLEL_LEVEL: 2
   CTEST_PARALLEL_LEVEL: 2
   CTEST_OUTPUT_ON_FAILURE: 1
-  CXXFLAGS: "-Wall -Wextra -Wpedantic"
+  CXXFLAGS: "-Wall -Wextra -Wpedantic -march=broadwell"
   QT_QPA_PLATFORM: "offscreen"
   QT_DEBUG_PLUGINS: 1
+  OPENBLAS_NUM_THREADS: 1
+  MKL_NUM_THREADS: 1
```

Comment:
Is this needed with MKL_THREADING sequential?

Reply:
Probably not, but let's play it safe.